src/sas/qtgui/Utilities/Preferences/PreferencesWidget.py

```diff
@@ -28,7 +28,15 @@ def set_config_value(attr: str, value: Any, dtype: Optional[Callable] = None):
         # The only way to get here **should** be during development, thus the debug log.
         logger.debug(f"Please add {attr} to the configuration and give it a sensible default value.")
 
+def config_value_setter_generator(attr: str,dtype: Optional[Any] = None):
```

Comment:
This might be a duplicate function. If it is, they do the same thing, but only one is needed.

Reply:
It is!  Strange that git didn't pick up on it.  Removed the duplicate.